docs/source/building-applications.rst

```diff
@@ -16,6 +16,7 @@ To run the examples, you will first need to `install the Daml SDK <https://docs.
   app-dev/app-arch
   app-dev/bindings-ts/index
   json-api/index
+  bindings-java/index
```

Comment:
```suggestion
  app-dev/bindings-java/index
```
The build is failing, I believe the link in the original TOC tree was relative to this location.